### ISSUE TITLE
Sort the posting authors with Liquid.

### ISSUE THREAD

**Issue by itrich** [SovereignCloudStack/website#449]
See reference in

_Originally posted by @itrich in https://github.com/SovereignCloudStack/website/issues/448#issuecomment-1306795679_

**Comment by itrich** [comment]
Blocked by #444 as merging may get complicated. So let's wait for the open PR to be resolved.

**Comment by itrich** [comment]
Won't do as we'll switch to Docusaurus as blog posting engine
